specs/phase0/p2p-interface.md

```diff
@@ -1121,20 +1110,6 @@ On the other hand, TLS 1.3 is the newest, simplified iteration of TLS.
 Old, insecure, obsolete ciphers and algorithms have been removed, adopting Ed25519 as the sole ECDH key agreement function.
 Handshakes are faster, 1-RTT data is supported, and session resumption is a reality, amongst other features.
 
-### Multiplexing
-
-#### Why are we using mplex/yamux?
-
-[Yamux](https://github.com/hashicorp/yamux/blob/master/spec.md) is a multiplexer invented by Hashicorp that supports stream-level congestion control.
-Implementations exist in a limited set of languages, and it’s not a trivial piece to develop.
-
-Conscious of that, the libp2p community conceptualized [mplex](https://github.com/libp2p/specs/blob/master/mplex/README.md)
-as a simple, minimal multiplexer for usage with libp2p.
-It does not support stream-level congestion control and is subject to head-of-line blocking.
-
-Overlay multiplexers are not necessary with QUIC since the protocol provides native multiplexing,
-but they need to be layered atop TCP, WebSockets, and other transports that lack such support.
```

Comment:
I think this paragraph about QUIC is valuable. I think we shouldn't lose it. Maybe should we change the topic to "Why are we using yamux?" instead and remove the mplex part?

Reply:
A better question might be "why don't we go straight to QUIC and skip yamux as a requirement"

Reply:
> A better question might be "why don't we go straight to QUIC and skip yamux as a requirement"

It's hard to find a stable QUIC library for some programming languages and it's not easy to implement on your own (I guess).

Reply:
> A better question might be "why don't we go straight to QUIC and skip yamux as a requirement"

Yeah, and the answer is that it's not easy to find a stable QUIC library for some programming languages and we decide to make QUIC optional instead, but preferable.

Reply:
the same can be said of yamux which is libp2p-only - https://blog.cloudflare.com/http3-usage-one-year-on saw 30% quic traffic share last year, meaning that any language that doesn't have a quic will have one soon.

ie in nim-libp2p, we have a yamux that hasn't been battle-tested _really_ but it's kind of pointless doing it knowing that it's on its way out.

mplex otoh has been working for the consensus layer for years now so a few more months to get quic in place is not going to make any difference.